### PR TITLE
Chatbot: Lovelits by Kartika and Lisa 

### PR DESCRIPTION
**Pull request by katirum**

### CODE REVIEW

**Comment by MariaWellander** [comment]
Here's the link to the bot :D https://phenomenal-kelpie-2b24fb.netlify.app/

**Review by MariaWellander — COMMENTED** [review]
- Am I able to understand the code easily?
We are absolutely able to understand the code easily.
- Does the code have lots of duplicates? Could it be tidied up?
Maybe the code has duplicates and could get compressed but when reading the code we feel that it’s nice to be able to follow and understand every step.
- Does it work or does anything look broken?
	The code works and nothing looks broken - good job!
- Does it follow the general and blue requirements?
	We have read the iterations and the code met up to them.
- To sum it up:
We really like the bot. It’s beautiful and we can see that you have put effort in the visual user experience. The functionality is to the point.
//Linnéa Wahlström & Maria Wellander
